twake/frontend/src/app/app.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import LoginService from 'services/login/login.js';
+import Integration from 'scenes/Integration/integration.js';
+import RouterServices from './services/RouterServices';
+import { Switch, Route } from 'react-router-dom';
+import { Router } from 'react-router';
+import ErrorBoundary from 'app/scenes/Error/ErrorBoundary';
+import 'app/ui.scss';
+
+export default () => {
+  useEffect(() => {
+    LoginService.init();
+  }, []);
+
+  return (
+    <Integration>
+      <Router history={RouterServices.history}>
+        <Switch>
+          {RouterServices.routes.map((route: any, index: number) => {
+            return (
+              <Route
+                key={route.name + ' ' + index}
```

Comment:
Avoir string concatenation when possible, use string literals, it makes clean code